games/o.yaml

```diff
@@ -1172,6 +1172,23 @@
   type: clone
   url: https://opendungeons.github.io/
   updated: 2021-04-08
+  
+- name: OpenEnroth
+  originals:
+  - Might and Magic VI: The Mandate of Heaven
+  - Might and Magic VII: For Blood and Honor
+  - Might and Magic VIII: Day of the Destroyer
```

Comment:
this looks like invalid yaml
please validate this; you can also try adding the entry  using the forms, they produce valid yaml

Reply:
ah, quotation marks did the trick. Thank you :+1: